x-pack/plugin/eql/qa/rest/src/yamlRestTest/resources/rest-api-spec/test/eql/10_basic.yml

```diff
@@ -478,3 +506,68 @@ setup:
           query: 'sequence with maxspan=10d [network where user == "ADMIN"] ![network where used == "SYSTEM"]'
   - match: { error.root_cause.0.type: "verification_exception" }
   - match: { error.root_cause.0.reason: "Found 1 problem\nline 1:75: Unknown column [used], did you mean [user]?" }
+
```

Comment:
Or maybe in this or a new yml file to add the more varied scenarios for "failed shards" queries that I mentioned above.

Reply:
I added more cases to toml tests